contrib/podmanimage/stable/Dockerfile

```diff
@@ -11,9 +11,21 @@ FROM registry.fedoraproject.org/fedora:latest
 # Don't include container-selinux and remove
 # directories used by yum that are just taking
 # up space.
-RUN useradd podman; yum -y update; yum -y reinstall shadow-utils; yum -y install podman fuse-overlayfs --exclude container-selinux; rm -rf /var/cache /var/log/dnf* /var/log/yum.*
+RUN dnf -y update; yum -y reinstall shadow-utils; \
+yum -y install podman fuse-overlayfs --exclude container-selinux; \
+rm -rf /var/cache /var/log/dnf* /var/log/yum.*
 
-ADD https://raw.githubusercontent.com/containers/libpod/master/contrib/podmanimage/stable/containers.conf /etc/containers/
+RUN useradd podman; \
+echo podman:10000:5000 > /etc/subuid; \
+echo podman:10000:5000 > /etc/subgid;
+
```

Comment:
Just something to think about, not blocking for this PR but that we probably want to address in future.

With such ranges specified:
- If the outer Podman doesn't run in a user namespace, all the inner Podman will end up using the same IDs.
- If the outer Podman runs in a user namespace, there won't likely be enough IDs to satisfy this request.

If we want to address the first scenario, we could probably fix it at runtime, and have the entrypoint script generate these entries with a random initial ID.

Reply:
Yes I think in the long run we want to make this more flexible, some how, But for now, we want to get to the point that we have an image which people can experiment with.